CONTRIBUTING.md

```diff
@@ -0,0 +1,374 @@
+2. Generates `CHANGELOG.md` for each package to be released
+3. Bump the version number in the `package.json` for each package
+4. Creates a git tag `<package-name>@<version>` pointing to the new release
+5. Create a new GitHub release for each package
+6. Publishes the new version to the NPM registry
+
+### Version changes guidelines
+
+**semantic-release** uses the Pull request title to determine the consumer impact of changes in the codebase.
+Following formalized conventions for Pull request title messages, **semantic-release** automatically determines the next [semantic version](https://semver.org) number, generates a changelog, and publishes the release. Please read the [pull request message format](#pull-request-message-format) section for more details.
+
+The table below shows which Pull Request titles get you which release:
+
+| Release type                         | Type and scope that triggers it                                                                                     | Commit message example                                      |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| ~~Patch~~ `vX.Y.🆙` Fix Release      | Type must be `fix`, `docs`, `refactor`, `style`, `revert` or `chore(deps)` (in this case the scope must match)      | `fix(topology): fix pipelinerun status icon color`          |
+| ~~Minor~~ `vX.🆙.0` Feature Release  | Type must be `feat`, `perf`                                                                                         | `feat(kiali): add namespace selector`                       |
+| ~~Major~~ `v🆙.0.0` Breaking Release | `<type>(<optional-scope>)!: <message>` Notice the `!` token. It must be present. The type and scope are irrelevant. | `feat(ocm)!: Relocate OCM config and support multiple hubs` |
+| No release                           | Type `chore` (except when the scope is `deps`), or for any type when the scope is `no-release`                      |
+
+## Creating new plugins
+
+A particular case of contribution is implementing a new plugin. Before you start implementing a new one, please consult our issue tracker and make a [New plugin request](#new-plugin-request). Once you have a green light from the community, you can start implementing your plugin.
+
+Start by running our plugin template wizard:
+
+```bash
+yarn new
+```
+
+This will lead you through plugin category selection and set up all the required boilerplate. By default, this will only create a new standalone plugin within the `plugins` folder and will not touch the `packages` folder where we host an empty Backstage instance. You can bypass this behavior by running `janus-cli new` directly instead, but we discourage it. We aim to enable a standalone, self-contained plugin development workspace, however, this is not fully possible at this moment.
```

Comment:
I think you can remove two commas from the last sentence "...standalone self-contained plugin development workspace, however this is not fully..."